packages/components/src/components/text-field/text-field.css

```diff
@@ -302,7 +302,8 @@ input:-webkit-autofill,
 input:-webkit-autofill:hover,
 input:-webkit-autofill:focus,
 input:-webkit-autofill:active {
-  color-scheme: dark;
+  /* hack to remove autocomplete colors in chrome, as they conflict with scale dark/light mode styles https://stackoverflow.com/a/68240841 */
```

Comment:
I'm not super happy about this haha… do you think we could try testing with adding `color-scheme: light dark;` globally in :root (in the global CSS file) and remove this `color-scheme` rules for the Text Field only?

Reply:
hey I did test that already in the old PR for this same issue https://github.com/telekom/scale/pull/2191#discussion_r1384904000 , but just adding `color-scheme: light dark;` to the global css does not fix the issue as far as i can see... (these changes would still be needed for it to work https://github.com/telekom/scale/pull/2191/files)

Reply:
i mean, it seems like just adding `color-scheme: light dark;` by itself would not fix the issue